eo-maven-plugin/README.md

```diff
@@ -124,4 +124,9 @@ Maven option `-U` (see [Maven CLI docs](https://maven.apache.org/ref/3.1.0/maven
 mvn -U clean install
 ```
 
+If you want to build your project or run your tests which use eo-maven-plugin, you can change stack size for your purposes by stack-size option:
 
+```
+mvn clean install -Pqulice -D.stack-size=1M
```

Comment:
@ivan-egorov42 why do we need to introduce new property? 
Can't we just run ```mvn clean install -Pqulice -Xss1M```?
If I'm not mistaken Maven will recognise it.

Reply:
got it - it was made to introduce increased default value

Reply:
`-Xss` is jvm flag, so we need to give it to surefire plugin by options of `mvn`